packages/lisk-transactions/test/utils/get_transaction_bytes.ts

```diff
@@ -125,17 +125,6 @@ describe('getTransactionBytes module', () => {
 				);
 			});
 
-			it('should not throw on type 0 with an amount that is 0', () => {
```

Comment:
should we check type 0 with amount 0 throws an error?

Reply:
I think it's not necessary since this check already exists in 0_transfer tests, and is outside the scope of this func.